pkg/services/ngalert/notifier/channels/util.go

```diff
@@ -317,3 +317,53 @@ func splitCommaDelimitedString(str string) []string {
 	}
 	return res
 }
+
+// Copied from https://github.com/prometheus/alertmanager/blob/main/notify/util.go, please remove once we're on-par with upstream.
+// truncationMarker is the character used to represent a truncation.
+const truncationMarker = "…"
+
+// Copied from https://github.com/prometheus/alertmanager/blob/main/notify/util.go, please remove once we're on-par with upstream.
+// TruncateInrunes truncates a string to fit the given size in Runes.
+func TruncateInRunes(s string, n int) (string, bool) {
+	r := []rune(s)
+	if len(r) <= n {
+		return s, false
+	}
+
+	if n <= 3 {
+		return string(r[:n]), true
+	}
+
+	return string(r[:n-1]) + truncationMarker, true
+}
+
+// TruncateInBytes truncates a string to fit the given size in Bytes.
+// TODO: This is more advanced than the upstream's TruncateInBytes. We should consider upstreaming this, and removing it from here.
```

Comment:
Copied verbatim from `webex.go`. I added the TODO so we remember to do this in the future.